js/utility/toggleButtonDisplayByAttributes.js

```diff
@@ -0,0 +1,9 @@
+export default function toggleButtonDisplayByAttributes(element, ...attrs) {
+  attrs.forEach((attr) => {
+    if (element.querySelector(attr).style.display === "none") {
+      element.querySelector(attr).style.display = "inline-block";
+    } else {
+      element.querySelector(attr).style.display = "none";
```

Comment:
statement in branch `else` does similar as the `if` condition. why is it required to override the display property with same value?

Reply:
I have used this `if else` statement to toggle the value of` style.display.` while it is `inline-block` then it will go to the `else `statement and if none then it will go to `if` statement.